pkgs/development/dhall-modules/prelude/v3.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  name = "dhall-prelude";
+
+  src = fetchFromGitHub {
+    owner = "dhall-lang";
+    repo = "Prelude";
+    rev = "v3.0.0";
+    sha256 = "1mdpylyd3bs5zx292c851fcl6cpghqrmnbdkxi5gn07vd6fvbaa6";
+  };
+
+  phases = [ "unpackPhase" "installPhase" ];
+
+  installPhase = ''
+    cp -r . $out
+  '';
+
+  meta = {
+    # TODO: https://github.com/dhall-lang/Prelude/issues/13
```

Comment:
This should be unblocked now.  `dhall-lang` now has a BSD 3 license:

https://github.com/dhall-lang/dhall-lang/pull/264